src/pages/family/[id].tsx

```diff
@@ -80,6 +87,12 @@ const FamilyPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({ pa
     <Layout title={page.title}>
       <section className="mb-8">
         <DocumentHead document={page} onCollectionClick={handleCollectionClick} />
+        <div className="bg-gray-200 border-b border-lineBorder py-4">
+          <div className="container">
+            Family/document from search or local store: <br />{" "}
+            <code className="block whitespace-pre-wrap">{JSON.stringify(familyResult, null, "\t")}</code>
```

Comment:
I'm guessing this is left in deliberately for the moment?

Reply:
Yeh its just the UI for the family page getting it ready to work with the search results